src/main/java/com/techiteasy/demo/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,65 @@
+package com.techiteasy.demo.controllers;
+
+import com.techiteasy.demo.exceptions.RecordNotFoundException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.*;
+
+//Stap 2: voorzie de klasse van de juiste annotatie
+@RestController
+@RequestMapping
+
+//Deze klasse bevat alle gevraagde CRUD-endpoints
+public class TelevisionsController {
+
+    //Deze mock is een tijdelijke oplossing om data in het geheugen op te slaan, zodat de functionaliteit van de controller getest kan worden zonder een database of externe opslag te gebruiken.
+    private final List<String> televisionDataBase = new ArrayList<>(Arrays.asList("Samsung", "Sony", "LG")); //Ik heb dit 'final' gemaakt om duidelijk te maken dat de referentie naar het Map-object onveranderlijk is na initiele toewijzing (oa voor betere leesbaarheid van de code en het evt. per ongeluk her-toewijzen van het veld elders in de code).
```

Comment:
Heel handig om je lijst alvast met data te initialiseren. Je beschrijft ook een hele goede reden om je list final te maken. Heel goed dat je daar over nagedacht hebt!

Reply:
Je comment spreekt nog van een Map, maar ik neem aan dat dat een voorgaande implementatie was en je dat aangepast hebt naar List?

Reply:
Deze "database" is niet meer nodig, nu we met een echte database verbinding kunnen make